tests/utils.go

```diff
@@ -504,16 +501,12 @@ func MountCloudInitFunc(devName string) func(*v1.VirtualMachineInstance) {
 func RunVMAndExpectLaunchWithRunStrategy(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine, runStrategy v1.VirtualMachineRunStrategy) *v1.VirtualMachine {
 	By("Starting the VirtualMachine")
 
-	Eventually(func() error {
-		updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		updatedVM.Spec.Running = nil
-		updatedVM.Spec.RunStrategy = &runStrategy
-		_, err = virtClient.VirtualMachine(updatedVM.Namespace).Update(context.Background(), updatedVM, metav1.UpdateOptions{})
-		return err
-	}, 300*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
-
-	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+	p, err := patch.New(
+		patch.WithAdd("/spec/running", nil),
+		patch.WithAdd("/spec/runStrategy", &runStrategy),
+	).GeneratePayload()
+	Expect(err).NotTo(HaveOccurred())
+	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, p, metav1.PatchOptions{})
```

Comment:
I'm not sure if this refactor is worth the effort, given that there are only few remaining calls for this function, and both are expected to be removed soon:

[PR #12359](https://github.com/kubevirt/kubevirt/pull/12359)
[PR #12137](https://github.com/kubevirt/kubevirt/pull/12137)

Reply:
Not that dramatic as it's already here. If the others are merged before, then we'll drop it anyway.